lib/Doctrine/Common/Annotations/CachedReader.php

```diff
@@ -247,8 +267,7 @@ private function getLastModification(ReflectionClass $class)
     }
 
     /**
-     * @param ReflectionClass $reflectionTrait
-     * @return int
+     * @return int|false
```

Comment:
wrong phpdoc. `max` return `false` only for empty arrays, and we always pass a non-empty array.

Reply:
I wasn't so sure but if you are, I will change it.

Reply:
well, there is a second case where it can return `false`, but that's when you give it an array of boolean.
The signature of `max` would be described like that:

- `max(empty-array): false` (assuming a non-existent `empty-array` type)
- `max(non-empty-array<T>): T` (using the existent `non-empty-array` type of Psalm)